ios/brave-ios/Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -594,21 +594,24 @@ public class PortfolioStore: ObservableObject, WalletObserverStore {
       guard !Task.isCancelled else { return }
       // fetch price for every token
       let allTokens = allVisibleUserAssets.flatMap(\.tokens)
-      let allAssetRatioIds = allTokens.map(\.assetRatioId)
-      let prices: [String: String] = await assetRatioService.fetchPrices(
-        for: allAssetRatioIds,
-        toAssets: [currencyFormatter.currencyCode],
-        timeframe: timeframe
+      let prices: [BraveWallet.AssetPrice] = await assetRatioService.fetchPrices(
+        for: allTokens.map {
+          BraveWallet.AssetPriceRequest(
+            coinType: $0.coin,
+            chainId: $0.chainId,
+            address: $0.contractAddress.isEmpty ? nil : $0.contractAddress
+          )
+        },
+        vsCurrency: currencyFormatter.currencyCode
       )
-      for (key, value) in prices {  // update cached values
-        self.pricesCache[key] = value
-      }
+
+      self.pricesCache.update(with: prices)
 
       // fetch price history for every non-zero balance token
       let nonZeroBalanceAssetRatioIds: [String] =
         allTokens
         .filter { (tokenBalancesCache[$0.id] ?? [:]).values.reduce(0, +) > 0 }
-        .map { $0.assetRatioId }
+        .map { $0.assetRatioId.lowercased() }
```

Comment:
`assetRatioId` is a computed value we created initially to pick from `coingeckId`, `symbol` and `contractAddress`. does this still stay valid with this PR's change?

Reply:
I think it's still valid. This piece of code is for price history, which still relies on the Ratios service and coingecko IDs for lookups.